crawl-ref/source/dat/des/arrival/simple.des

```diff
@@ -2029,3 +2029,80 @@ x.G.G.G.x
 x.......x
 xxxx+xxxx
 ENDMAP
+
+###############################################################################
+
+NAME:   giann_horns   
+TAGS:   arrival no_monster_gen
+ORIENT: float
+MAP
+x@xx...................xx@x
+x..xx.................xx..x
+x...xx..G..xx+xx..G..xx...x
+x....xx...xx...xx...xx....x
+x.....xxxxx.....xxxxx.....x
+x............{............x
+xx.......................xx
+ xxxxxxxxxxxx+xxxxxxxxxxxx
+ENDMAP
+
+NAME:   giann_bridges
+TAGS:   arrival no_monster_gen
+MONS:   plant
+ORIENT: float
+MAP
+.........................
+.1wwwwwwww1...1wwwwwwww1.
+.wwwWWWWwww...wwwWWWWwww.
+.wwwWWWWwww...wwwWWWWwww.
+.1wwwwwwww.....wwwwwwww1.
+............{............
+.1wwwwwwww.....wwwwwwww1.
+.wwwWWWWwww...wwwWWWWwww.
+.wwwWWWWwww...wwwWWWWwww.
+.1wwwwwwww1...1wwwwwwww1.
+.........................
+ENDMAP
+
+NAME:   giann_guarded_fountains
+TAGS:   arrival no_monster_gen no_item_gen no_rotate
+MONS:   hobgoblin
```

Comment:
Ouch, this is brutal. Make the player fight two hobgiblins as their first encounter, with no escape route? This will kill a lot of players. The decor looks nice but I don't like the mechanics.

Also I think you should replace most of the `.` floor tiles in the top half with ` ` empty tiles, telling the dungeon builder it's okay to put walls or whatever there. Having a huge open area like this is scary for the player, who relies on terrain to create favourable encounters.

Personally I'd remove the hobgoblins and the whole upper half of the map, and just have the lower half for some nice decor. If you're married to the "guarded" theme, I suggest some less formidable guards.